src/main/java/org/apache/commons/io/output/DeferredFileOutputStream.java

```diff
@@ -272,4 +274,31 @@ public void writeTo(final OutputStream outputStream) throws IOException {
             }
         }
     }
+
+    /**
+     * Gets the current contents of this byte stream as an {@link InputStream}.
+     * If the data for this output stream has been retained in memory, the
+     * returned stream is backed by buffers of {@code this} stream,
+     * avoiding memory allocation and copy, thus saving space and time.<br>
+     * Otherwise, the returned stream will be one that is created from the data
+     * that has been committed to disk.
+     *
+     * @return the current contents of this output stream.
+     * @throws IOException if this stream is not yet closed or an error occurs.
+     * @see org.apache.commons.io.output.ByteArrayOutputStream#toInputStream()
```

Comment:
Needs a Javadoc since tag.

Reply:
Ping. FYI, you should run the _default_ Maven build: 'mvn'.

Reply:
Added `@since`.

I tried running the default maven build, but it doesn't work for me. It always dies in the middle with a "killed" message on the terminal. I did run all the individual goal that are part of the default build and they all complete successfully.